README.md

```diff
@@ -8,6 +8,11 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 $ yarn
 ```
 
+If you are using MacOS, run this after yarn installation:
+```
+yarn add docusaurus --dev
```

Comment:
The website is developed usin `npm` instead of `yarn` -- so please update your fork because I've just updated the README file to reflect this. You should use `npm install` and `npm run start`.

Also, you should not include changes like this within a pull request that is about something else.

Reply:
remove and merged with latest in main.